src/app/cli/src/init/graphql_internal.ml

```diff
@@ -163,7 +164,16 @@ struct
   let execute_query ctx schema variables operation_name query =
     match Graphql_parser.parse query with
     | Ok doc ->
-        Schema.execute schema ctx ?variables ?operation_name doc
+        let open Async in
+        Monitor.try_with (fun () ->
```

Comment:
We have to use a Monitor.try_with because `try .. with` doesn't work with Async. 

https://github.com/MinaProtocol/mina/issues/18254